internal/alpini/netty4/alpini-netty4-base/src/main/java/com/linkedin/alpini/netty4/ssl/SslInitializer.java

```diff
@@ -307,6 +313,10 @@ public void userEventTriggered(ChannelHandlerContext ctx, Object evt) throws Exc
                 }
               }
             }
+          } catch (SSLPeerUnverifiedException ex) {
+            if (needClientAuth) {
+              LOG.warn("Unable to obtain remote CN for {}", ctx.channel().remoteAddress(), ex);
```

Comment:
I wonder if we can simplify this slightly.  It occurs to me that if needClientAuth is true and we have an unverified principal in the session request, then the request was likely a failure as I would expect the sslEngine to have kicked in and enforced the rule.

So instead of doing the above chunk where we get the clientAuth/sslparameters/etc. we could probably just do 

`if (!succeed) {
  LOG.warn(....
}`

since we already spent the time getting that information and the log would help us out to puzzle a failure since it would correlate with the failed request.